### PR TITLE
docs: add command page for `env deploy`

### PR DESCRIPTION
**Pull request by Lou1415926**
As well as adding a version number beside the title.

Screen Shot 2022-08-10 at 4 54 00 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -69,6 +69,7 @@ nav:
         - app upgrade: docs/commands/app-upgrade.en.md
         - app delete: docs/commands/app-delete.en.md
         - env init: docs/commands/env-init.en.md
+        - env deploy: docs/commands/env-deploy.en.md
```
Comment (efekarakus) [inline mkdocs.yml]: 🤔 should our `deploy` commands be under `Release`?

**Review by efekarakus — APPROVED** [review]

**Review by huanjani — APPROVED** [review]
Yay for version! Looks good. Adding DNM label in case you want to move deploy(s) to Release.

File: mkdocs.yml (hunk 1 above)
Comment (Lou1415926) [reply mkdocs.yml]: I can see arguments for both! It feels to me that people run `deploy` commands in an earlier stage when they are setting up the infrastructure - to test out the infra, to know about Copilot, etc. Hence `Build`.  After this stage they use some CICD (GitHub actions, CodePipelines... etc.) to "release" their application changes, hence `Release`.

**Pull request merged**